### ISSUE TITLE
Video speed and pitch

### ISSUE THREAD

**Issue by Ozilook** [Frontesque/VueTube#108]
It could be useful to add a dropdown menù button to choose between different video speeds and pitches (also customized).

**Comment by tired6488** [comment]
Similar to issue #16. Are customizable video pitches a useful feature?

**Comment by Ozilook** [comment]
> Similar to issue #16. Are customizable video pitches a useful feature?

In a certain way, yes.

Think of all those videos with wrong pitch, applied to avoid problems with copyright. 🤷‍♂️

**Comment by Svk1190** [comment]
Many music players have an inbuilt equalizer, or functionality to modify pitch, or functionality to change the left/right balance of stereo music. However, neither the YouTube app nor the YouTube music app provides such functionalities. 
For some people, these functionalities are a medical necessity, for example: 
1) Aged people cannot hear high frequency components properly. Hence an equalizer that amplifies the high-frequency components would be very beneficial. 
2) People with different hearing loss in two ears might prefer to modify the left/right balance of stereo music. 

Hence, it would be helpful to include an equalizer in VueTube. The below images provide some examples of equalizer, available in some Android players. 

![image](https://user-images.githubusercontent.com/12389625/159142810-26db2214-dd60-4304-8321-b42b14b0adb1.png)

**Comment by rakibdev** [comment]
Yes. As someone who use YouTube only for music This pitch changer totally needed when a music has broken deep voice and want to make it sound like female tone.
